apiconcepts/filetypesupport/create_a_new_bil_project.md

```diff
@@ -6,9 +6,9 @@ In this chapter you will learn how to properly set up a project for developing a
 Create the Project
 --
 
-After launching <Var:VisualStudioEdition> choose to create a new <Var:ProductName> Plug-in Project, and give it an appropriate name, e.g. Sdl.Sdk.FileTypeSupport.Samples.WordArtVerifier. The instructions for creating a <Var:ProductName> Plug-in Project are described in the **Plug-in Framework SDK** under **<Var:ProductName>** > **Building a Plug-in**
+After launching <Var:VisualStudioEdition> choose to create a new <Var:ProductName> Plug-in Project, and give it an appropriate name, e.g. Sdl.Sdk.FileTypeSupport.Samples.WordArtVerifier. The instructions for creating a <Var:ProductName> Plug-in Project are described in the [**Building a Plug-in**](\..\\..\articles\gettingstarted\building_a_plugin.md) article under the **Getting Started** section.
```

Comment:
does the link work? i see a \...\\..which I do not think it works

Reply:
It's working locally, I intentionally used that to get to the md file located in another folder